types/hl7/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "npm-naming": false,
+        "no-useless-files": false
```

Comment:
I'm a little unsure about why these are setting (given that it seems to match an existing npm module?)

Reply:
This package is standalone, it types HL7 Standard version 2.5 which is an annotation like CSV or JSON widely used in Healthcare, you can find more info [here](https://www.hl7.org/)